web-console/4-patterns/masthead.md

```diff
@@ -29,6 +31,32 @@ On the landing page, the only navigation available is the Primary Masthead.
 
 ## Implementation Details
 
+### User
+
+#### Regions
+- Users will have the ability to view and change their cluster region.
+
+![Full Masthead](img/Region.png)
+- For MVP, the cluster region will be listed as a fly-out option under the user menu in the masthead.  
+- The current region will be displayed in the dropdown.
+
+#### Set Home Page
+- Users will have the ability to view and change their home page. This setting will determine what page will be displayed upon initial login.
+
+![Full Masthead](img/setHome.png)
+- A menu option called "Set Home Page" will be displayed under the user dropdown, underneath the Regions (if present).
+
+![Full Masthead](img/setHome-2.png)
+- Clicking Set Home Page will launch a modal with the following three options:
+	- Catalog Home (Default)
+	- My Projects List
+	- Project Overview
```

Comment:
I'm unsure if Project Overview is the best label for this, would it be better to say Selected Project?  You may have already iterated on this, but just want to verify

Reply:
@serenamarie125 I'd be fine with changing it and agree it could be more clear! I was thinking we should tell the user the actual page they would land on (Overview) and was hoping the dropdown below the radio button would help clarify that you actually have to select a project (default text in the dropdown is "Select project"). @cshinn @ncameronbritt Any preference on naming for the third radio button?

Reply:
What about something like Overview for [Select Project v]?

I agree that Project Overview is potentially confusing here, and Selected Project sounds like it should select the project you're currently on.

Reply:
Good idea...something like this work?

![screen shot 2017-09-07 at 10 58 48 am](https://user-images.githubusercontent.com/18057044/30169902-9ff667ba-93bb-11e7-87c7-3ade4414a001.png)

Thoughts @ncameronbritt @serenamarie125

Reply:
That seems clearer to me.